src/test/java/components/ui/LoginReportPortalTest.java

```diff
@@ -0,0 +1,23 @@
+package components.ui;
+
+import com.ui.LoginReportPortalPage;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import utilities.ReadPropertyFile;
+
+import java.io.IOException;
+
+public class LoginReportPortalTest  extends BaseTest {
+
+    ReadPropertyFile fr = new ReadPropertyFile();
+
+
+
+
+    public void openTheReportUrl(WebDriver driver) throws IOException {
+        LoginReportPortalPage loginReportPortalPage = new LoginReportPortalPage(driver);
```

Comment:
Move test(s) object(s) initialization to the BaseTest class, name it setUp and anotate with @BeforeClass or @BeforeSuite. And this class should represent the tests that related to the Login page of Report portal service.
You can move the test from TestNGRunnerTest to this class with anotations @Test.